src/gluonts/model/simple_feedforward/_network.py

```diff
@@ -158,15 +158,24 @@ def hybrid_forward(
             distr_args, loc=loc, scale=scale
         )
 
-        # (batch_size, prediction_length, target_dim)
-        loss = distr.loss(future_target)
+        if isinstance(distr, NanMixture):
+            # demasking the missing values for the future_targets
```

Comment:
I agree with you that maybe these changes don’t belong in this PR. The results look great and everything else looks fine, so maybe you can remove the changes here and then we can merge!

Reply:
Thanks! I removed the changes.